pulsar-client-api/src/main/java/org/apache/pulsar/client/api/schema/GenericRecord.java

```diff
@@ -82,7 +83,8 @@ default SchemaType getSchemaType() {
      * @return the internal representation of the record
      * @throws UnsupportedOperationException if the operation is not supported
      */
-    default Object getNativeRecord() {
+    @Override
+    default Object getNativeObject() {
```

Comment:
This is a BC break change to an API that is already marked as stable.  Why do we need to change the name?

Reply:
@jerrypeng this is a new method introduced a couple of weeks ago with https://github.com/apache/pulsar/pull/9614

we are free to change it now, and it is better to do it now, that we have not released 2.8 yet